docs/00-Getting-started/02-Overview.md

```diff
@@ -0,0 +1,128 @@
+
+This straightforward approach allows you to quickly incorporate widgets into your project. Refer to the component documentation for usage details. Currently, AgnosUI supports Angular, React, and Svelte.
+
+- use the core functionality to create your own customized widget.
+
+This approach offers complete customization of widgets, including markup and styling, while benefiting from AgnosUI's headless capabilities for managing display and interactions.
+
+This method requires a deeper understanding and more effort, detailed in the comprehensive documentation. You can start building your widget based on existing widget code.
+
+## Motivation
+
+Maintaining full widget libraries over time can be challenging due to framework changes, shifts in usage patterns, and projects adopting different frameworks. AgnosUI addresses this challenge by separating the state and behavior of a widget from its rendering engine (the frameworks). This separation brings stability and several benefits:
+
+- Core functionalities are managed in pure TypeScript, avoiding issues tied to UI frameworks' constraints.
+- The main code stays the same even if the framework changes.
+- Fixes or new features seamlessly apply to all supported frameworks.
+- Unit tests are straightforward, focusing on the core written in pure Typescript for enduring reliability.
+- UI Frameworks are used only rendering the UI based on the widget state.
+- Consistency is maintained across frameworks, ensuring uniformity for organizations using different frameworks.
+
+## How does it work ?
+
+The basic idea is to maage a widget from a data point of view only and apis to interact with these data. This is a global picture:
+
+The fundamental concept of AgnosUI revolves around managing a widget purely from a data perspective, along with APIs to interact with this data. The high-level workflow is illustrated below:
+
+![svg](../../demo/src/resources/images/overview-workflow.svg)
+
+AgnosUI constructs widgets based on data, independent of the rendering engine in use. To be framework-agnostic, the system must provide:
+
+- All necessary data for the UI.
+- APIs to modify this data.
+- A reactive system to notify the rendering engine of data changes, enabling synchronized UI updates.
+
+## Reactivity with Tansu
+
+The reactivity in AgnosUI is managed with [Tansu](https://github.com/AmadeusITGroup/tansu).
+
+Tansu has been initially developed by following the Svelte store specifications. It then extends its features to achieve full store management capabilities:
+
+- Supports all [Svelte stores specifications](https://svelte.dev/docs/svelte-store),
+- The `Signal` capabilities have been added. Notably, `computed` is available along with `derived`. The store itself is a function that we can call to get the instant value.
+- Provides a `batch` method to prevent glitch issues.
+- Ensures synchronous subscriber calls, preventing unpredictable issues in integration with other frameworks.
```

Comment:
Synchronous subscriber calls are ensured to prevent unpredictable issues when integrating with other frameworks.